.github/ISSUE_TEMPLATE/Standard-Bug-Issue-Template.md

```diff
@@ -27,31 +27,42 @@ labels: Bug
 ## Notes
 < bulleted list of special notes or suggestions to the development team on any information pertinent to the bug. > 
 
-## Definition of Ready
-
+## Ready for Ready to Work
 - [ ] The Issue maintains standard format for Issues and meets INVEST
-- [ ] Story Defined and written appropriately
-- [ ] Steps to Recreate supplied and sufficiently detailed
-- [ ] Expected and Actual Behavior supplied
-- [ ] Business Value supplied
+- [ ] Story and Business Value complete
```

Comment:
for Business value, seems like it would just be the same Business Value of the original features. Should it maybe be a description of the effected features, or where the workflow is halted?

Reply:
If we have an issue for the original feature. But really, there is a "cost" associated with the system not behaving as expected. It can be expressed and we can decide how to prioritize these behavior changes. Remember that in most cases, workflow isn't halted and the issue is worked around. That work around is one of the costs associated with the bug and the business value is avoiding that cost. But it could be trivial effort, like ignoring a missing "." on the last sentence of a message. Technically a bug but not anything we would prioritize.